about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css" />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/71d90399a2.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>capstone- about</title>
+</head>
+<body class="about">
+  <header>
+    <nav class="topnavbar">
+      <div></div>
+      <ul class="topnavbaricons">
+        <li><i class="fab fa-facebook-f"></i></li>
+        <li><i class="fab fa-twitter"></i></li>
+        <li>english</li>
+        <li>my page</li>
+        <li>logout</li>
+      </ul>
+    </nav>
+    <nav id="secnavbar" class="secnavbar">
```

Comment:
- [ ] As requested by the former code reviewer, which also a requirement for this project, please make sure you can switch between the home and about page in you mobile view. 
- [OPTIONAL ] Please kindly make sure your mobile view is centralized, the footer is also absent for the mobile view, please kindly fix that.  see image below.  
![TIMO01](https://user-images.githubusercontent.com/65852621/137288486-737c6288-a51f-4b8d-99a7-68213b95a338.png)
 
![TIMO02](https://user-images.githubusercontent.com/65852621/137288527-43deaed4-9c4c-4569-9c43-26d79ff15a5d.png)